content/blog/eclipse-adoptium-projects-ready-for-community-review/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Eclipse Adoptium Projects Ready for Community Review
+date: "2020-12-21T12:00:00+00:00"
+author: tsc
+description: Community Review is the last step before we can officially start Eclipse Adoptium. It gives all of you an opportunity to ask questions, suggest improvements, or raise concerns. We also have an update to our transition roadmap.
+tags:
+  - AdoptOpenJDK
+  - Eclipse
+  - Foundation
+---
+
+Six months ago, we announced that AdoptOpenJDK is going to join the Eclipse Foundation and rebrand as Eclipse Adoptium. After a long period of work behind the scenes, the first batch of project proposals is ready for community review. At the Eclipse Foundation, the monolithic AdoptOpenJDK project will be split into multiple sub-projects under the umbrella of the [Eclipse Adoptium Top Level Project](https://projects.eclipse.org/projects/adoptium):
+
+* **Eclipse AQAvit** encompasses all our activities around testing, especially the development of our [AQA test suite](/2019/07/the-first-drop-introducing-adoptopenjdk-quality-assurance-aqa-v1-0/).
+* **Eclipse Temurin** (an anagram of "runtime") focuses on building and releasing JDKs (that part that most associate with AdoptOpenJDK).
+
+AdoptOpenJDK's application to become an Eclipse project was approved under the provision that we only release binaries that have passed the respective TCK. The [TCK for Java SE](https://openjdk.java.net/groups/conformance/JckAccess/) is a proprietary technology of Oracle and to be kept secret -- something that is in stark contrast to the spirit of open source. The Eclipse Foundation and AdoptOpenJDK went to great lengths to reconcile Oracle's wishes with the realities of open source. The result is the [Eclipse Temurin Compliance](https://projects.eclipse.org/proposals/eclipse-temurin-compliance) project, which governs access to TCK material and resources. Thanks to the Eclipse Temurin Compliance project, most contributors can continue their work without being impacted by the TCK requirement. We thank Eclipse Foundation's Executive Director Mike Milinkovich and his team for reaching an agreement with Oracle and making this possible.
```

Comment:
Probably need to remove `something that is in stark contrast to the spirit of open source`

Reply:
And change `The Eclipse Foundation and AdoptOpenJDK went to great lengths to reconcile Oracle's wishes with the realities of open source` to `The Eclipse Foundation and AdoptOpenJDK worked at length with Oracle to arrange access` or something like that.

Reply:
I concur. We should find more neutral words.

Reply:
+1 to simply deleting the portion Martijn highlights.  Your explanation stands on its own without it.  All owners have the right **not** to make their code available under an open source license.

Reply:
The words are true and accurate.  If you wish to remove the emotion and human aspect from them, then perhaps...  

"and to be kept secret -- while antithetical to the spirit of open source, nonetheless a requirement.  The Eclipse Foundation and AdoptOpenJDK took time and undertook the necessary discourse to reconcile Oracle's requirement with the realities of open source."